GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Kroxylicious Governance
+
+This document outlines the governance model for Kroxylicious.
+This governance model is designed to uphold the principles of transparency, open collaboration, and community involvement.
+
+[cfc]: https://www.commonhaus.org/bylaws/cf-council.html
+[cdm]: https://www.commonhaus.org/bylaws/decision-making.html
+[ctp]: https://www.commonhaus.org/policies/trademark-policy/
+[coc]: ./CODE_OF_CONDUCT.md
+[contrib]: ./CONTRIBUTING.md
+
+## Table of Contents
+
+- [Roles and Responsibilities](#roles-and-responsibilities)
+- [Decision Making](#decision-making)
+- [Code of Conduct](#code-of-conduct)
+- [Trademark Policy](#trademark-policy)
+- [Contributing](#contributing)
+
+## Roles and Responsibilities
+
+- **Code owners:** Individuals with write access to the repository, responsible for reviewing and merging contributions, and steering project direction. These individuals (or groups) will be maintained in the repository (`CODEOWNERS` or `.github/CODEOWNERS`).
+- **Contributors:** Anyone who contributes to the project in any form.
+
+Reference: [About code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
+
+## Decision Making
+
+This project follows The [Commonhaus decision making][cdm] process.
```

Comment:
This should be a look to the main Commonhaus repo right?

Reply:
Yes, it does.  That's a reference to a URL defined higher up.